src/cmake/axom-config.cmake.in

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+@PACKAGE_INIT@
+
+if(NOT AXOM_FOUND)
+
+  #----------------------------------------------------------------------------
+  # Set version and paths
+  #----------------------------------------------------------------------------
+  
+  set(AXOM_VERSION "@AXOM_VERSION_FULL@")
```

Comment:
We should also publish the `MAJOR`, `MINOR` and `PATCH` variables since we have those. That way consumers of this can add conditional logic in their respective CMake and or BLT build system.